### PR TITLE
Nicer readme suggestion

### PR DESCRIPTION
**Pull request by jej**
What this change is adding / fixing
-----------------------------------
Just some cosmetic on the github home page...

screenshot_20221021-012254

### CODE REVIEW

**Comment by jej** [comment]
screenshot_20221021-012254

**Comment by jej** [comment]
Ok thanks, I will investigate there. Github removes the styles and most of html tags, so I will look at the hacks available... I wanted to avoid a new image to do that, but maybe it's better to create a specific one for this homepage.

**Comment by jej** [comment]
Not sure it's possible to align image and text. Strawberry use a hack. It's not their logo it's a `:strawberry:` emoticon... The padding between image and title is the h2 padding. Github manages...

File: README.rst [hunk 1]
```diff
@@ -17,13 +17,18 @@
     :alt: Translation status
     :target: https://hosted.weblate.org/engage/quodlibet/
 
+.. raw:: html
+
+    <p></p>
+    <img src="https://github.com/quodlibet/quodlibet/raw/master/quodlibet/images/hicolor/256x256/apps/io.github.quodlibet.QuodLibet.png" width="104px" alt="Quod Libet logo" />
+
```
Comment (declension) [inline README.rst:24]:
```suggestion

.. image:: ./quodlibet/images/hicolor/scalable/apps/io.github.quodlibet.QuodLibet.svg
   :width: 128px
   :alt: Quod Libet logo
   :align: left

```

**Comment by declension** [comment]
> Not sure it's possible to align image and text. Strawberry use a hack. It's not their logo it's a `:strawberry:` emoticon... The padding between image and title is the h2 padding. Github manages...

Managed it a different way. Thanks!

**Comment by jej** [comment]
Cool thanks

**Pull request merged**